src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -90,32 +90,37 @@ We recommend linking to the [explanation of default groups](/docs/accounts/accou
 
 ## General philosophy for user permissions/roles language [#user-language-philosophy]
 
-Our general philosophy is summed up in these points:
+First, it will help you to understand the relationship between groups, roles, and capabilities in the new New Relic One user model (sometimes referred to as v2 users). We recommend reading [this explanation of how these concepts relate](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#type-role-group). 
 
-### Document role-related requirements
+The following explanations about our permissions documentation philosophy applies for documenting our New Relic One user model, not our original user model.
```

Comment:
This sentence feels a little confusing to me... 

I prefer "concepts" to "explanations" here. I feel like the use of "explanations" sort of muddies the waters here.

Maybe it could be simplified to:

"The following concepts relate to our New Relic One user model permissions, not our original user model."